test/aws/osml/models/aircraft/test_aircraft_model.py

```diff
@@ -36,8 +36,7 @@ def test_predict_aircraft_model(self):
         data_binary = open("assets/images/2_planes.tiff", "rb")
         response = self.client.post("/invocations", data=data_binary)
 
-        assert response.status_code == 200
-
+        # assert response.status_code == 200
```

Comment:
What status code is being returned? Why not check?

Reply:
I commented out when testing other things and forgot to uncomment - will revert